app/src/main/java/com/activity_sync/screens/MyProfileScreen.java

```diff
@@ -0,0 +1,143 @@
+package com.activity_sync.screens;
+
+import android.content.Context;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.activity_sync.App;
+import com.activity_sync.R;
+import com.activity_sync.presentation.models.User;
+import com.activity_sync.presentation.presenters.IPresenter;
+import com.activity_sync.presentation.presenters.MyProfilePresenter;
+import com.activity_sync.presentation.services.CurrentUser;
+import com.activity_sync.presentation.services.INavigator;
+import com.activity_sync.presentation.views.IMyProfileView;
+import com.activity_sync.utils.CredibilityService;
+import com.amulyakhare.textdrawable.TextDrawable;
+
+import javax.inject.Inject;
+
+import butterknife.Bind;
+import rx.Observable;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.android.view.ViewObservable;
+
+public class MyProfileScreen extends ScreenWithMenu implements IMyProfileView
```

Comment:
There are couple repetitions from UserDeatilsScreen + maybe normal Screen not ScreenWithMenu